src/utils.jl

```diff
@@ -17,7 +17,15 @@ function get_admin_credentials()::Tuple{String, String}
         pwd = ""
     end
 
-    return (usr, pwd)
+    # Token
+    if haskey(ENV, "MOS_ADMIN_TKN")
+        tkn = ENV["MOS_ADMIN_TKN"]
```

Comment:
Let's use MOS_BACKEND_TOKEN for consistency with python compute.
Let's also update the readme under "Environment Variables":
* MOS_ADMIN_USR and MOS_ADMIN_PWD, or MOS_BACKEND_TOKEN

Reply:
Yes, MOS_BACKEND_TOKEN used in [numerous places](https://github.com/search?q=org%3AFuinn+MOS_BACKEND_TOKEN&type=code)